packages/sonarwhal/tests/helpers/rule-runner.ts

```diff
@@ -11,55 +11,87 @@ import { ids as connectors } from './connectors';
 import { createServer } from './test-server';
 import { IRuleConstructor, RulesConfigObject } from '../../src/lib/types';
 import * as resourceLoader from '../../src/lib/utils/resource-loader';
-import { RuleTest } from './rule-test-type';
+import { RuleTest, RuleLocalTest } from './rule-test-type';
 import { Sonarwhal } from '../../src/lib/sonarwhal';
 import { SonarwhalConfig } from '../../src/lib/config';
+import { getAsUri } from '../../src/lib/utils/get-as-uri';
 
-/** Executes all the tests from `ruleTests` in the rule whose id is `ruleId` */
-export const testRule = (ruleId: string, ruleTests: Array<RuleTest>, configs: { [key: string]: any } = {}) => {
-
-    /**
-     * Creates a valid sonarwhal configuration. Eventually we should
-     * test all available connectors and not only JSDOM
-     */
-    const createConfig = (id: string, connector: string, opts?): SonarwhalConfig => {
-        const rules: RulesConfigObject = {};
+// Regex to replace all scenarios: `http(s)://localhost/`, `http(s)://localhost:3000/`
+const localhostRegex = /(http|https):\/\/localhost[:]*[0-9]*\//g;
 
-        if (opts && opts.ruleOptions) {
-            rules[id] = ['error', opts.ruleOptions];
-        } else {
-            rules[id] = 'error';
-        }
+/**
+ * Creates a valid sonarwhal configuration. Eventually we should
+ * test all available connectors and not only JSDOM
```

Comment:
Can you remove the last phrase ("Eventually we should..."). It doesn't apply anymore (although we are not testing `local`.